crd-convert/src/main/java/io/strimzi/kafka/crd/convert/utils/ContentType.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.utils;
+
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public enum ContentType {
+    JSON("application/json", new JsonMapper(), "json", "jsn"),
+    YAML("application/x-yaml", new JsonMapper(new YAMLFactory()), "yaml", "yml");
```

Comment:
Why not a `YAMLMapper`?

Reply:
I guess it does the same, will change it.